_includes/tutorials/aggregating-average/kstreams/markup/dev/make-topology.adoc

```diff
@@ -0,0 +1,12 @@
+////
+In this file you describe the Kafka streams topology, and should cover the main points of the tutorial.
+The text assumes a method buildTopology exists and constructs the Kafka Streams application.  Feel free to modify the text below to suit your needs.
+////
+
+Then create the following file at `src/main/java/io/confluent/developer/RunningAverage.java`. Let's take a close look at the `buildTopology()` method, which uses the Kafka Streams DSL.
+
+// Full topology description goes here
+
```

Comment:
To help the reader get the main point of the tutorial may consider adding something like this 
with a brief description of the code

``` 
Here's how you'll calculate the running average:
+++++
<pre class="snippet"><code class="java">
   final KTable<Long, CountAndSum> ratingCountAndSum =
        ratingsById.aggregate(() -> new CountAndSum(0L, 0.0),
                              (key, value, aggregate) -> {
                                aggregate.setCount(aggregate.getCount() + 1);
                                aggregate.setSum(aggregate.getSum() + value);
                                return aggregate;
                              },
                              Materialized.with(Long(), countAndSumSerde));
</code></pre>
+++++
```

Just a suggestion though

Reply:
done 👍